lints/lint_dnsname_contains_question_marks.go

```diff
@@ -0,0 +1,46 @@
+package lints
+
+import (
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/util"
+	"strings"
+)
+
+type DNSNameContainsQuestionMarks struct{}
+
+func (l *DNSNameContainsQuestionMarks) Initialize() error {
+	return nil
+}
+
+func (l *DNSNameContainsQuestionMarks) CheckApplies(c *x509.Certificate) bool {
```

Comment:
This probably only applies to subscriber certificates.

Reply:
done